api/net/Areas/Editor/Controllers/AVOverviewController.cs

```diff
@@ -23,10 +23,10 @@ namespace TNO.API.Areas.Editor.Controllers;
 [ApiController]
 [Area("editor")]
 [ApiVersion("1.0")]
-[Route("api/v{version:apiVersion}/[area]/reports/av/overviews")]
```

Comment:
Don't bother renaming the API endpoint path.

Reply:
Updated, thanks